edx_sysadmin/management/commands/tests/test_git_add_course.py

```diff
@@ -38,16 +37,17 @@ class TestGitAddCourse(SharedModuleStoreTestCase):
     Tests the git_add_course management command for proper functions.
     """
 
-    TEST_REPO = "https://github.com/edx/edx4edx_lite.git"
-    TEST_COURSE = "MITx/edx4edx/edx4edx"
-    TEST_BRANCH = "testing_do_not_delete"
-    TEST_BRANCH_COURSE = CourseKey.from_string("MITx/edx4edx_branch/edx4edx")
-
     ENABLED_CACHES = ["default", "mongo_metadata_inheritance", "loc_cache"]
 
     def setUp(self):
         super().setUp()
         self.git_repo_dir = settings.GIT_REPO_DIR
+        self.TEST_REPO = "https://github.com/edx/edx4edx_lite.git"
+        self.TEST_COURSE = self.store.make_course_key("MITx", "edx4edx", "edx4edx")
```

Comment:
Since we are creating the course key right here, a better name might be `TEST_COURSE_KEY` just for better readability.